features/step_definitions/home_navigation.feature

```diff
@@ -0,0 +1,11 @@
+Feature: Be able to navigate to other pages from home page
+
+  Scenario: User wants to view evaluation data
+    Given User is on the home page
+    When User clicks on the evaluation data button
+    Then User should see the evaluations page
+
+  Scenario: User wants to view faculty member historical data
+    Given User is on the home page
+    When User clicks on the faculty member historical data button
+    Then User should see the faculty member historical data page
```

Comment:
WIll do.  Is it generally good practice to always have an empty line at the end?